scripts/spack/packages/axom/package.py

```diff
@@ -369,25 +376,20 @@ def initconfig_hardware_entries(self):
 
             hip_link_flags = ""
 
-            rocm_root = os.path.dirname(spec["llvm-amdgpu"].prefix)
+            rocm_root = spec["llvm-amdgpu"].prefix
```

Comment:
I thought this required a removal of the `llvm` directory in the prefix of `llvm-amdgpu`...

Reply:
hmm... this won't work because Axom's CI is using the predefined host-configs.

Reply:
So I’ll just push the fix in the spack configs along with the removal of ROCM_ROOT_DIR.
How should I test those changes? I'm thinking you might want to trigger host-config generation when there are changes in the local spack configs and packages.

Reply:
we generate and test our spack recipes manually then check in the generated host-config.